### ISSUE TITLE
[Supporting Platform - Bot Framework Web Chat - Card Breakfast]: In 200% zoom mode 'White-labelled' control is truncated.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5362]
### Is it an issue related to Adaptive Cards?

No.

### User Experience: 
If content is getting truncated or not visible at 200% zoom, then users with low vision who depend on this feature will not be able to identify these controls and perform actions on them.
### Note:
User credentials should not be included in the bug.
### Pre-requisite:
Achieve 200% Zoom in Browser Zoom.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify whether the controls are truncated in 200% zoom mode or not.

### What do you expect?

 In 200% zoom mode 'White-labelled' control should not get truncated. All the controls should be clearly visible.

### What actually happened?

 In 200% zoom mode 'White-labelled' control is truncated.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/77fbfb0a-1977-4953-b15a-ea8c90f95e89)
![Image](https://github.com/user-attachments/assets/80537e3f-ef8a-49d4-aed8-4544d8cad3b9)

### MAS reference

MAS 1.4.4 – Resize Text

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev3;#HCL;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.4.4;#Win11-Edge(Chromium);
